### ISSUE TITLE
Unable to prepare

### ISSUE THREAD

**Issue by cppcooper** [InfosecSapper/keepnote#10]
Maybe I'm just doing things wrong, and well I do hate python..
I can't get the environment prepared for development it seems. One of the readme files indicates you must run `make dev` from the project root, but this hasn't been working for me. I tried from upstream master, and both of your branches.

```
$ sudo make dev
[Mon 12:17:42]
[sudo] password for josh: 
  File "<string>", line 1
    import keepnote; print keepnote.PROGRAM_VERSION_TEXT
                           ^
SyntaxError: invalid syntax
virtualenv --system-site-packages env
make: virtualenv: No such file or directory
make: *** [Makefile:71: env/bin/activate] Error 127
```

All I'm trying to do is get this into a working state so I can update the libffi usage, or whatever is gonna need doing so that I can run keepnote again. I'm running archlinux, and I'm stuck with an updated libffi due to dependencies. I've opened an issue about that here https://github.com/mdrasmus/keepnote/issues/754

**Comment by henshin** [comment]
You are probably using Python 3, no? This is an old app, so you can only use Python 2.6 to compile it.
I would recommend to use some other note-taking application as this one is not being maintained any more. Be prepared for some headaches if you decide to keep going and try to update libraries on this. I've been through that

**Comment by cppcooper** [comment]
Yep, python 3. Tried installing other versions, but hadn't thought to replace the `/usr/bin/python` link and have it point to 2.7 until today, but even still I am unable to `make dev`. With a git error seemingly blocking my path
```
npm ERR! code 128
npm ERR! An unknown git error occurred
npm ERR! command git --no-replace-objects ls-remote ssh://git@github.com/cppcooper/jest.git
npm ERR! git@github.com: Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-10-26T16_42_28_744Z-debug.log
```
I went as far as to fork the original repo, so I could use my ssh key to clone it via ssh (even though I added the https url). So no idea why it fails.

Yea, honestly [since yesterday] I've been thinking it might be easier to write my own from scratch in C++ and learn Qt so I can use that for a simple gui. I did like the structure of this one so figured using something similar, just without the sql lite (*eyeroll*), with simple directories and files would be a good idea. Something so simple, shouldn't take long at all to finish.. but then I've gotta name it which would take me forever.. I wonder if I could just use keepnote :S v2 perhaps

**Comment by henshin** [comment]
it won't work with Python 2.7. I only managed to compile it with Python 2.6 and it was still a bit of a headache.

I've moved on from Keepnote and started using Typora and Mark Text. I miss some of the features of Keepnote, like the custom ordering, icons on the directory structures, etc but once you start using these kind of WYSIWYG markdown editors, you will never go back.

**Comment by cppcooper** [comment]
Those don't look like they'd accomplish what I was using keepnote for, easily.

I might try python25 later on, but I really think I'm gonna just be making my own from scratch. Just not totally sure when I'll get to it.. and definitely unsure what I'll name it.
